containers/dotnetcore-mssql/.devcontainer/mssql/postCreateCommand.sh

```diff
@@ -0,0 +1,80 @@
+dacpac="false"
+sqlfiles="false"
+SApassword=$1
+dacpath=$2
+sqlpath=$3
+
+echo "SELECT * FROM SYS.DATABASES" | dd of=testsqlconnection.sql
+for i in {1..60};
+do
+    /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P $SApassword -d master -i testsqlconnection.sql
+    if [ $? -eq 0 ]
+    then
+        echo "sql server ready"
+        break
+    else
+        echo "not ready yet..."
+        sleep 1
+    fi
+done
+rm testsqlconnection.sql
+
+for f in $dacpath/*
+do
+    if [ $f == $dacpath/*".dacpac" ]
+    then
+        dacpac="true"
+        echo "found dacpac $f"
+    fi
+done
+
+for f in $sqlpath/*
+do
+    if [ $f == $sqlpath/*".sql" ]
+    then
+        sqlfiles="true"
+        echo "found sql file $f"
+    fi
+done
+
+if [ $sqlfiles == "true" ]
+then
+    echo "installing mssql-tools"
+    curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+    curl https://packages.microsoft.com/config/ubuntu/18.04/prod.list | tee /etc/apt/sources.list.d/msprod.list
```

Comment:
The .NET containers are about to update to 20.04, so it would be good not to hard code the version.  You can avoid this problem by using the following instead:

```bash
DISTRO=$(lsb_release -is | tr '[:upper:]' '[:lower:]')
CODENAME=$(lsb_release -cs)
echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-${DISTRO}-${CODENAME}-prod ${CODENAME} main" > /etc/apt/sources.list.d/microsoft.list
```

Reply:
thanks for the code sample - bash is not my strength